README.md

```diff
@@ -190,30 +192,25 @@ Further delegation of a capability with `nb` fields set MUST respect the `nb` fi
 ``` json
 [
   {
-    "with": "example://example.com/public/photos/",
-    "can": "crud/delete"
-  },
-  {
-    "with": "example://example.com/private/84MZ7aqwKn7sNiMGsSbaxsEa6EPnQLoKYbXByxNBrCEr",
-    "can": "wnfs/append"
-  },
-  {
-    "with": "example://example.com/public/photos/",
-    "can": "crud/delete",
-    "nb": {
-      "matching": "/(?i)(\W|^)(baloney|darn|drat|fooey|gosh\sdarnit|heck)(\W|$)/"
+    "example://example.com/public/photos/": {
+      "crud/delete": {},
```

Comment:
```suggestion
      "crud/read": {},
```

I think having duplicate keys keys in the example is probably not a good idea, especially if we're saying we don't want to support it.

Reply:
Oh haha I think this is an actual typo from a copy/paste